src/Stratis.Bitcoin.Features.RPC/RPCClient.Wallet.cs

```diff
@@ -557,10 +554,8 @@ public void WalletPassphrase(string passphrase, int timeout)
         /// <param name="timeout">Timeout in seconds</param>
         public async Task WalletPassphraseAsync(string passphrase, int timeout)
         {
-            var parameters = new List<object>();
-            parameters.Add(passphrase);
-            parameters.Add(timeout);
-            await SendCommandAsync(RPCOperations.walletpassphrase, parameters.ToArray()).ConfigureAwait(false);
+            var parameters = new object[] {passphrase, timeout};
+            await SendCommandAsync(RPCOperations.walletpassphrase, parameters).ConfigureAwait(false);
```

Comment:
You might as well just inline this so that you don't need `parameters`:

`await SendCommandAsync(RPCOperations.walletpassphrase, new object[] {passphrase, timeout}).ConfigureAwait(false);`